sdk/servicebus/azure-servicebus/migration_guide.md

```diff
@@ -64,6 +64,11 @@ semantics with the sender or receiver lifetime.
 | `AutoLockRenew().register(queue_client.get_receiver(session='foo'))`| `AutoLockRenew().register(sb_client.get_queue_session_receiver(session_id='foo').session)`| [Access a session and ensure its lock is auto-renewed](./samples/sync_samples/session_send_receive.py) |
 | `receiver.get_session_state()` | `receiver.session.get_session_state()` | [Perform session specific operations on a receiver](./samples/sync_samples/session_send_receive.py)
 
+### Working with messages
+| In v0.50 | Equivalent in v7 | Note |
+|---|---|---|
+| `message.locked_until < datetime.now()`| `message.locked_until_utc < datetime.utcnow()`| All datetimes are now UTC and named as such|
+
```

Comment:
I would prefer the title of section being call something like "Working with time",
and then you could mention now all the time related parameter/variable/property are following "*_utc" naming pattern and taking utc time.

Reply:
So I do make the mention you say in the note to the right; the only reason I organized this as such is that we've been classifying these sections by "object" up to this point, and this gave a good conceptual grouping for if we do other things in the future (e.g. difference of peek message, if any are relevant?)

Now that said, I can see the value in your suggestion, since I believe we did the same tweaks for session (I'm trying to remember where we altered timestamps outside of message, since if it's broad, that's even more justification to do it your way.)  

I'll adjust it, this is such a small thing I wouldn't want to bikeshed us further and your structure already has one obvious advantage on mine.